ingestion_tools/dataset_configs/10436.yaml

```diff
@@ -1,379 +1,463 @@
-          ORCID: 0000-0003-0188-9799
-        - name: Damian C. Ekiert
-          ORCID: 0000-0002-2570-0404
-          corresponding_author_status: true
-        - name: Gira Bhabha
-          ORCID: 0000-0003-0624-6178
-          corresponding_author_status: true
-      cell_strain:
-        name: ATCC-50506
-      cell_type:
-        name: fungal spore
-        id: CL:0002369
-      cross_references:
-        publications: 10.1101/2024.07.13.603322
-        #TODO: Add EMPIAR-12176 once available
-        related_database_entries: EMD-45674, EMD-45671, EMD-45672, EMD-45673
-      dataset_description: This dataset contains raw frames, tilt series, tomogram reconstructions and segmentations from
-        Encephalitozoon intestinalis microsporidian spores. Samples were cryo-FIB milled.
-      dataset_identifier: 10436
-      dataset_title: Cryo-ET datasets (1 and 2) of dormant microporidian spores from Encephalitozoon intestinalis
-      dates: &id002
-        deposition_date: '2024-07-09'
-        last_modified_date: '2024-07-24'
-        release_date: '2024-07-29'
-      funding:
-        - funding_agency_name: American Heart Association
-          grant_id: '915749'
-        - funding_agency_name: Searle Scholars Program
-          grant_id: 'SSP-2018-2737'
-        - funding_agency_name: NIH/NIAID
-          grant_id: 'R01AI147131'
-        - funding_agency_name: The Pew Charitable Trusts
-          grant_id: 'PEW-00033055'
-        - funding_agency_name: NIH Common Fund Transformative High Resolution Cryo-Electron Microscopy program
-          grant_id: 'U24 GM129539'
-        - funding_agency_name: Simons Foundation
-          grant_id: "SF349247"
-        - funding_agency_name: NY State Assembly
-        - funding_agency_name: NIH/NIGMS
-          grant_id: "9 P41 GM103310"
-        - funding_agency_name: Irma T. Hirschl Career Scientist Award
-      grid_preparation: quantifoil 2/2 200mesh, ~20nm evaporated carbon, prepared using waffle method
-      organism:
-        name: Encephalitozoon intestinalis
-        taxonomy_id: 58839
-      other_setup: Cryo-FIB milling
-      sample_preparation: "E. intestinalis spores were purified from infected vero cells, then ran over a discontinuous
-      percoll gradient to separate out mature spores. For details, (see manuscript: Cryo-ET reveals the in situ
-      architecture of the polar tube invasion apparatus from microsporidian parasites)"
-      sample_type: cell
-    sources:
-      - literal:
-          value:
-            - '10436'
+alignments:
+- metadata:
+    format: IMOD
+  sources:
+  - source_multi_glob:
+      list_globs:
+      - Data_Set_2/{run_name}/aligned_tilt_series/*_IMOD/*.xf
+      - Data_Set_2/{run_name}/aligned_tilt_series/*_IMOD/*.tlt
+      - Data_Set_2/{run_name}/aligned_tilt_series/*_IMOD/*.com
+- metadata:
+    format: ARETOMO3
+  sources:
+  - source_multi_glob:
+      list_globs:
+      - Data_Set_2/{run_name}/aligned_tilt_series/*.st.aln
```

Comment:
- @uermel should we create two different alignment entries when there are both aln files and xf files?

Reply:
This should not occur within the same deposition.